receiver.go

```diff
@@ -80,6 +81,17 @@ func (r *Receiver) HandleMessage(ctx context.Context, handle func(*Message) erro
 	}
 }
 
+// AddCredit adds credits to the current link.
+func (r *Receiver) AddCredit(credit uint32) error {
```

Comment:
Is this what the JS implementation calls this?  .NET calls it "IssueCredit".

Reply:
Yes, this is what rhea calls it. 

I'm juggling this in my head - 'IssueCredit' makes it sound like it's actually making the credits happen _now_. In reality AddCredit is merely adding it to the 'pending' credits (ie, the actual 'issuing' happens later).

Does this distinction make much sense to you? I've got no real preference either way, just want to think it over a bit with you.

Reply:
I don't really have a preference for either, I just want to ensure that if there's standard terminology to be used here that we abide by that (and I'm assuming that .NET, being the reference implementation, got this right).  Looking at the spec, there is one place where it uses the phrase "issuing link-credit" (figure 2.45 in section 2.6.9) so maybe that's why .NET chose "IssueCredit".

Reply:
I'll go with IssueCredit :)

Reply:
Done.